cfgov/unprocessed/css/molecules/multi-select.less

```diff
@@ -80,27 +80,26 @@
     }
 
     &_choices {
+        margin-right: unit( -10px / @base-font-size-px, em );
```

Comment:
Can you leave a note that this is necessary for when two choices are side by side? That wasn't immediately obvious to me till I tested locally.